src/components/message/message.pw.tsx

```diff
@@ -0,0 +1,333 @@
+        backgroundColor
+      );
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check ${children} as children for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<Message>{children}</Message>);
+
+      await expect(messageChildren(page)).toHaveText(children);
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check ${className} as className for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent className={className} />);
+
+      await containsClass(messagePreview(page), className);
+    });
+  });
+
+  testData.forEach((id) => {
+    test(`should check ${id} as id for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent id={id} />);
+
+      await expect(messagePreview(page)).toHaveId(id);
+    });
+  });
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when open is ${boolVal} for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent open={boolVal} />);
+      if (boolVal === true) {
+        await expect(messagePreview(page)).toBeVisible();
+      } else {
+        await expect(messagePreview(page).getAttribute("disabled")).toBeTruthy;
+      }
+    });
+  });
+
+  test(`should focus component when open is true and component has a ref`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<MessageComponentWithRef />);
+
+    await buttonPreview(page).click();
+    await expect(messagePreview(page)).toBeFocused();
+  });
+
+  testData.forEach((title) => {
+    test(`should check ${title} as title for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent title={title} />);
+      await expect(messageTitle(page)).toHaveText(title);
+    });
+  });
+
+  ([
+    [true, "rgba(0, 0, 0, 0)"],
+    [false, "rgb(255, 255, 255)"],
+  ] as [boolean, string][]).forEach(([boolVal, backgroundColor]) => {
+    test(`should apply expected styling when transparent is ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent transparent={boolVal} />);
+
+      await expect(messagePreview(page)).toHaveCSS(
+        "background-color",
+        backgroundColor
+      );
+    });
+  });
+
+  ([
+    [true, 50],
+    [false, 20],
+  ] as [boolean, number][]).forEach(([boolVal, paddingVal]) => {
+    test(`should check showCloseIcon when it's ${boolVal} for Message component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent showCloseIcon={boolVal} />);
+
+      if (boolVal === true) {
+        await expect(messageDismissIcon(page)).toBeVisible();
+        await expect(messageContent(page)).toHaveCSS(
+          "padding",
+          `15px ${paddingVal}px 15px 20px`
+        );
+      } else {
+        await expect(messageDismissIcon(page).getAttribute("disabled"))
```

Comment:
Same suggestion as above, as the discussion about ".toBeTruthy" function is a bit flaky, would you be able to change it to ".not.toBeVisible();"